src/renderer/features/governance/model/unlock/unlock-validate-model.ts

```diff
@@ -0,0 +1,102 @@
+import { type ApiPromise } from '@polkadot/api';
+import { type SignerOptions } from '@polkadot/api/submittable/types';
+import { type Store, createEffect, createEvent, sample } from 'effector';
+import { combineEvents } from 'patronum';
+
+import { type Asset, type Balance, type Chain, type ID, type Transaction } from '@shared/core';
+import { toAccountId, transferableAmount } from '@shared/lib/utils';
+import { balanceModel, balanceUtils } from '@entities/balance';
+import { networkModel } from '@entities/network';
+import { transactionService } from '@entities/transaction';
+// TODO: fix it after DDD refactoring
+import {
+  type AmountFeeStore,
+  type ValidationResult,
+  validationUtils,
+} from '@/features/operations/OperationsValidation';
```

Comment:
```suggestion
} from '@/features/operations';
```

Reply:
there's no import from `operations` itself for these files